TESTS/network/wifi/mbed_app.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
Thanks for pointing it out. Not the first time I checked in wifi password. Really wish we do something about it in Mbed OS to prevent it from happening again.